tools/test-all

```diff
@@ -10,8 +10,8 @@ eval set -- "$TEMP"
 # extract options.
 while true; do
     case "$1" in
-        -f | --force)
-            FORCEARG="--force"
+        -f | --skip-provision-check)
```

Comment:
We should probably in a separate commit remove the `-f` alias here, since it doesn't super make sense with the new variable name.  I'm not aware of anyone using this feature and had forgotten it existed.